docs/api/dt/dt.rst

```diff
@@ -0,0 +1,20 @@
+
+.. xdata:: datatable.dt
+	:src: src/datatable/__init__.py dt
+
+	This is the :mod:`datatable` module itself.
+
+	The purpose of exporting this symbol is so that you can easily
+	import all the things you need from the datatable module in one
+	go::
```

Comment:
Don't we need to use `.. code-block:: python` here?

Reply:
Double-colon at the end of a paragraph is a shortcut for `.. code-block:: python` (and it also leaves one `:` in the rendered html).